packages/vcvrack-sonusmodular/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_slug=SonusModular
+_name=sonusmodular
+pkgname=vcvrack-sonusmodular
+pkgver=2.0.0
+pkgrel=1
+pkgdesc='VCV Rack plugin collection'
+arch=(x86_64 aarch64)
+url='https://gitlab.com/sonusdept/sonusmodular'
+license=(GPL3)
+groups=(proaudio vcvrack-plugins)
+depends=(vcvrack)
+makedepends=(simde zstd)
+source=("$pkgname-$pkgver.zip::https://gitlab.com/sonusdept/$_name/-/archive/$pkgver/$_name-$pkgver.zip")
+sha256sums=('1168cd4276d70d87ac4843fb3875953ae532f4c356ecd88002ccfbb9ee3d10be')
+
+build() {
+  cd $_name-$pkgver
+  # vcvrack make flags
+  export SLUG=$_slug
+  export VERSION=$pkgver
+  export RACK_DIR=/usr/share/vcvrack
+  make dist
+}
+
+package() {
+  cd $_name-$pkgver
+  install -d "$pkgdir"/usr/lib/vcvrack/plugins
+  cp -va dist/$_slug -t "$pkgdir"/usr/lib/vcvrack/plugins
```

Comment:
Do we need to install:

* the font licenses
* the Windows VST(3) DLLs
* the `LICENSE.txt` file in the plugin dir?

Reply:
Removed the DLLs in 0b27b4cb5b6a55b689989804f1a399f8fd7ad563.

Regarding license files, not sure, the original VCV Library releases include the files in the package ... But maybe we should go and move them to `/usr/share/licenses/$pkgname` or in case of a common license, remove it. I'll think about it.

Reply:
I decided to put the font licenses to `/usr/share/licenses/$pkgname` (as they're not common licenses) but discard the GPLv3 license files.